src/Apps/Conversation/Components/NoMessages.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button, Flex, Sans } from "@artsy/palette"
+import { RouterLink } from "Artsy/Router/RouterLink"
+import React from "react"
+
+export const NoMessages = () => {
+  return (
+    <Flex
+      flexDirection="column"
+      justifyContent="center"
+      alignItems="center"
+      height="100%"
+    >
+      <Sans size="6" weight="medium" mb={2}>
```

Comment:
I'm sure that this comes from the designs but the medium weight felt a bit strong to my eyes when paired with the bold "Inbox" header at the top.

Reply:
Appreciate the feedback! Will definitely pass that along. We've got a little while before this launches so visual tweaks are still very likely.